src/prefect/client/schemas/objects.py

```diff
@@ -506,6 +506,9 @@ def populate_deprecated_fields(cls, values: Any):
         return values
 
 
+KeyValueLabels = Dict[str, Union[StrictBool, StrictInt, StrictFloat, str]]
```

Comment:
I believe this is already defined in this file https://github.com/PrefectHQ/prefect/blob/main/src/prefect/client/schemas/objects.py#L74

Reply:
ah. so pickett beat me huh 😄 .

fixed up - good catch thx! 🫡